### DISCUSSION TITLE
Search for user

### DISCUSSION THREAD

**Discussion by TzeroOcne** [SevenTV/chatterino7#280] [General]
Is there a way to search for user in chat like the one in twitch chat like this
![image](https://github.com/user-attachments/assets/36fe654f-b6ef-4d0e-a7b0-e80e2d462024)

**Comment by Nerixyz** [comment]
This is only possible if you're a moderator or a broadcaster in a channel, because of limitations from Twitch.

![](https://github.com/user-attachments/assets/72400fdb-a147-4e8e-a93e-b84cbead3b73)
